splunk-otel-android/src/main/java/com/splunk/rum/SplunkRumBuilder.java

```diff
@@ -240,16 +240,30 @@ public SplunkRumBuilder setDeploymentEnvironment(String environment) {
     }
 
     /**
-     * Configures span data filtering.
+     * Configures span data filtering. This method is deprecated in favor of configureSpanFilter().
      *
      * @param configurer A function that will configure the passed {@link SpanFilterBuilder}.
      * @return {@code this}
      */
+    @Deprecated
     public SplunkRumBuilder filterSpans(Consumer<SpanFilterBuilder> configurer) {
         configurer.accept(spanFilterBuilder);
         return this;
     }
 
+    /**
+     * Configures span data filtering.
+     *
+     * @param configurer A function that will configure the passed {@link
+     *     io.opentelemetry.rum.internal.SpanFilterBuilder}.
+     * @return {@code this}
+     */
+    public SplunkRumBuilder configureSpanFilter(
+            Consumer<io.opentelemetry.rum.internal.SpanFilterBuilder> configurer) {
```

Comment:
In general I think deprecating the old method and adding a new is a great idea, but I think we should probably wait a bit before we actually do that - at least until we upstream the OTel parts. If we add this as it is right now, we'll have our public API depend on the `io.opentelemetry.rum.internal` package.

Reply:
Yeah ok, fair enough...we don't want to have references to the internal package in our public API. I get it now. We're going to have to deal with the fact that EVERYTHING in otel is in internal at some point tho. 🤷🏻 

I removed the deprecation and removed the new configure method.